src/types/index.ts

```diff
@@ -611,3 +614,12 @@ export interface IDefaultComposableOptions {
    */
   store: Store<any>
 }
+
+export type WebSocketChannelTypes = 'Transactions' | 'MicroBlocks' | 'KeyBlocks';
```

Comment:
For the type that has the union of possible values please use singular name (same as for example `CurrencyCode`.
```suggestion
export type WebSocketChannelType = 'Transactions' | 'MicroBlocks' | 'KeyBlocks';
```